<launch>
        <node pkg="teleop_vision" type="robot_to_camera_aruco" name="robot_to_camera_aruco" output="screen">
 		<rosparam command="load" file="$(find teleop_vision)/launch/params_robot_camera_aruco.yaml" />
        	<param name="cam_intrinsic_calibration_file_path" value="$(find teleop_vision)/resources/ecml_camera_data.xml" />
         </node>

</launch>
